packages/agreement-email-sender/src/models/errors.ts

```diff
@@ -1,45 +1,62 @@
 import { DescriptorId, EServiceId, InternalError } from "pagopa-interop-models";
 
-export const errorCodes = {
-  selfcareIdNotFound: "0019",
-  eServiceNotFound: "0005",
-  institutionNotFound: "0024",
-  descriptorNotFound: "0014",
-};
+type AgreementEmailSenderErrorCode =
+  | "activationDateNotFound"
+  | "eServiceNotFound"
+  | "tenantNotFound"
+  | "tenantDigitalAddressNotFound"
+  | "descriptorNotFound";
```

Comment:
Making error codes an enum because these are internal errors, they are never converted to codes sent to the front-end as APIErrors